assignments/callbacks.js

```diff
@@ -27,25 +27,46 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, function(L) {
+  console.log(L);
+})
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[3]);
```

Comment:
While hardcoding 4 works in this case, the more general solution is `arr[arr.length-1]`